python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -274,7 +274,9 @@ def quantize(self, model,  # remove the type requirement for type checking
         """
         if not accelerator or accelerator == 'onnxruntime':
             # check if dataloader is of legal format
-            check_pytorch_dataloaders(model, [calib_dataloader, val_dataloader])
+            check_output_format = (metric is not None) and (val_dataloader is not None)
+            check_pytorch_dataloaders(model, [calib_dataloader, val_dataloader],
+                                      check_output_format=check_output_format)
```

Comment:
```suggestion
            check_pytorch_dataloaders(model, [calib_dataloader, val_dataloader],
                                      metric=metric)
```
Maybe use `if metric==None` as a switch to decide if we check y?